xray/services/enrich.go

```diff
@@ -27,7 +28,7 @@ func NewEnrichService(client *jfroghttpclient.JfrogHttpClient) *EnrichService {
 	return &EnrichService{client: client}
 }
 
-func (es *EnrichService) ImportGraph(importParams XrayGraphImportParams) (string, error) {
+func (es *EnrichService) ImportGraph(importParams XrayGraphImportParams, rootPath string) (string, error) {
```

Comment:
```suggestion
func (es *EnrichService) ImportGraph(importParams XrayGraphImportParams, fileName string) (string, error) {
```
I would change the name to match the param

Reply:
fixed